cattle/plugins/docker/compute.py

```diff
@@ -81,6 +82,16 @@ def _add_resources(self, ping, pong):
         if not utils.ping_include_resources(ping):
             return
 
```

Comment:
sorry, I hate to comment on small things like this.  But there would be less code if you did
```python
stats = None
try:
    stats = ....
except:
   log.exception(...)
```
You default the stats to None and only set it on success